regression/cbmc/Empty_struct2/main.c

```diff
@@ -0,0 +1,24 @@
+#include <assert.h>
+
+struct empty
+{
+};
+
+#pragma pack(1)
+struct S
+{
+  int x;
```

Comment:
If I was a complete sadist I might suggest making this an ```int8_t``` because then we have to deal with the packing rules around an empty structure.  If ```y``` was one as well you'd hope that ```sizeof(struct S)``` would be ```sizeof(int8_t) * 2``` rather than 2 (or 3) times the minimum alignment unit.